docs/release-notes/latest.md

```diff
@@ -4,174 +4,106 @@ append_help_link: true
 title: Latest release notes
 hide_title: true
 description: Release notes include the changes, fixes, and additions in specific versions of Semgrep.
+toc_max_heading_level: 3
+tags:
+  - Release notes
 ---
 
-# August 2023 release notes
+# September 2023 release notes
 
-## Semgrep OSS Engine
-
-:::caution
-Semgrep version 1.38.0 removed some features. This change may break your Semgrep workflows. See [Semgrep OSS > Removed](#removed) for more information.
+:::info
+* Moving forward, these release notes cover the following products:
+    * Semgrep Cloud Platform
+    * Semgrep Code
+    * Semrep Supply Chain
+    * Semgrep Assistant (beta)
+    * Semgrep documentation and knowledge base
+* Refer to **Semgrep OSS** release notes in [<i class="fas fa-external-link fa-xs"></i> Semgrep GitHub > Releases](https://github.com/returntocorp/semgrep/releases/) as the source of truth for OSS releases.
```

Comment:
Not directly related to this PR’s changes so this is food for thought and discussion elsewhere, but should we mirror these somewhere on the docs site, so that our search captures them and users don’t have to go over to GH to read them?

Reply:
Am open to that! To be clear, what I was trying to avoid was having one set of release notes worded slightly differently (as we used to do) from the ones in the releases. So if we were just pulling, we wouldn't run into the issue of 2 different SOTs.

Reply:
Yep, understood. Also to clarify, my comment was only related to the Semgrep OSS part, not lines 18-20.